modules/administration/pages/custom-channels.adoc

```diff
@@ -35,23 +37,29 @@ Do not create child channels containing packages that are not compatible with th
 
 
 .Procedure: Creating a Custom Channel
-
 . In the {productname} {webui}, navigate to menu:Software[Manage > Channels], and click btn:[Create Channel].
 . On the [guimenu]``Create Software Channel`` page, give your channel a name (for example, [systemitem]``My Tools SLES 15 SP1 x86_64``) and a label (for example, [systemitem]``my-tools-sles15sp1-x86_64``).
-Labels must not contain space characters or capital letters.
+Labels must not contain spaces or capital letters.
 . In the [guimenu]``Parent Channel`` drop down, choose the relevant base channel (for example, [systemitem]``SLE-Product-SLES15-SP1-Pool for x86_64``).
 Ensure that you choose the compatible parent channel for your packages.
 . In the [guimenu]``Architecture`` drop down, choose the appropriate hardware architecture (for example, [systemitem]``x86_64``).
 . Provide any additional information in the contact details, channel access control, and GPG fields, as required for your environment.
 . Click btn:[Create Channel].
 
 
+[IMPORTANT]
+====
+By default, the ``Enable GPG Check`` field is checked when you create a new channel.
+If you would like to add custom packages and applications to your channel, make sure you uncheck this field to be able to install unsigned packages.
+Keep in mind this is a security risk for packages from an untrusted source.
```

Comment:
Not sure whether we actually want sentence intros such as "Keep in mind".

Reply:
Oof, no I missed that one.